src/schematools/contrib/django/models.py

```diff
@@ -45,6 +45,7 @@ class DynamicModel(models.Model):
     #: Overwritten by subclasses / factory
     CREATION_COUNTER = None
     _dataset: Dataset = None  # type: ignore[assignment]
+    _dataset_schema: DatasetSchema = None  # type: ignore[assignment]
```

Comment:
Waar wordt deze geset?

Reply:
Deze werd al een lange tijd door `model_factory()` toegekend.

Reply:
Kunnen we 'm dan niet beter daar weghalen? Dit is weer een punt waarop inconsistenties binnen kunnen sluipen.